pkg/app/instances/docker.go

```diff
@@ -28,22 +31,24 @@ import (
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/api/types/image"
 	"github.com/docker/docker/client"
 )
 
 const DockerIMType IMType = "docker"
 
 type DockerIMConfig struct {
-	DockerImageName      string
+	DockerImageNames     []string
```

Comment:
add documentation for this new config. why having many images, which is supposed to be the one host needs to be created with?

Reply:
Done.

Reply:
I don't understand the documentation.

>If 2 or more images exist, former one would be chosen. 

If we chose the first one, why having a list of images?

> If no images exist, downloadable image with shortest latency would be chosen.

What do you mean with  "If no images exist"? Do you mean the list is empty or there's no relevant image in the local Docker Engine? Elaborate on what's a downloadable image.

Having a single bucket for image names, seems to make these logic complicated, feel free to add a new config parameter in order to make it easier to follow.

Reply:
Canceled to apply `DockerImageNames` configuration.